pf/tfbridge/resource_state.go

```diff
@@ -139,3 +157,24 @@ func parseTFSchemaVersion(m resource.PropertyMap) (int64, error) {
 	}
 	return 0, nil
 }
+
+func updateTFSchemaVersion(m resource.PropertyMap, version int64) (resource.PropertyMap, error) {
+	var meta map[string]interface{}
+	if metaProperty, hasMeta := m[metaKey]; hasMeta && metaProperty.IsString() {
+		if err := json.Unmarshal([]byte(metaProperty.StringValue()), &meta); err != nil {
+			err = fmt.Errorf("expected %q special property to be a JSON-marshalled string: %w",
+				metaKey, err)
+			return nil, err
+		}
+	} else {
+		meta = map[string]interface{}{}
+	}
+	meta["schema_version"] = version
```

Comment:
If we didn't have `updateTFSchemaVersion` before, what did we use `"schema_version"` for in the past? Is this the TF schema version, or a construct internal to the bridge?